### ISSUE TITLE
Ingest ALKIS-Parzellarkarte

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#155]
## User story
1. As a User
2. I want the Parzellarkarte (property lot map)
3. So that I can find the property associated with a specific kooerdinate/ address

## Acceptance criteria
* Display Parzellarkarte

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by op-hub** [comment]
Ingesting not necessary: data is only images (png)
Just display in FE via wmts service
